spectaql/TUTORIAL.md

```diff
@@ -48,7 +48,7 @@ curl --location --request POST $URI --header 'Content-Type: application/json' --
     "data": {
         "userLogin": {
             "errors": [],
-            "authToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI1ZjkwMTY3ODM2MmZmNjIzMjVkZmFmYjciLCJuZXR3b3JrIjoicmVndGVzdCIsImN1cnJlbmN5IjoiQlRDIiwiaWF0IjoxNjAzMjc4NDU2fQ.-im7fJS_aneM32zNXDVqSDPPoQtCtCE22X-hUclT3u0"
+            "authToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI1ZjkwMTY3ODM2MmZmNjIzMjVkZmFmYjciLCJuZXR3b3JrIjoicmVndGVzdCIsImN1cnJlbmN5IjoiQlRDIiwiaWF0IjoxNjAzMjc4NDU2fQ.-im7fJS_andM32zNXDVqSDPPoQtCtCE22X-hUclT3u0"
```

Comment:
is there a way to exclude this?

Reply:
I have investigated and you can exclude it via:
```
[default.extend-words]
ane = "ane"
```
but I'd prefer not to since `ane` is a legitimate typo.

We shouldn't be committing real tokens anyway so I don't see an issue with changing them. The signature will break but its just an example anyway. Someone following the tutorial must use their own token.